middleware/jwt.go

```diff
@@ -29,12 +29,12 @@ type (
 		// ErrorHandlerWithContext is almost identical to ErrorHandler, but it's passed the current context.
 		ErrorHandlerWithContext JWTErrorHandlerWithContext
 
-		// Signing key to validate token. Used as fallback if SigningKeys has length 0.
-		// Required. This or SigningKeys.
+		// Signing key to validate token. Used as fallback if KeyFunc is nil or SigningKeys has length 0.
+		// Required. This or SigningKeys or KeyFunc.
```

Comment:
@lammel : Do you have any suggestion to improve the documentation here? I feel that it is poorly written at this moment.

Basically, the order of priority is KeyFunc, SigningKeys and SigningKey. Only one of them is required. They are kind of mutually exclusive, but there is no error given to the user/dev when they specify more than one.

Reply:
Looking good now. Very nice!